app/yaba.yaml

```diff
@@ -22,6 +22,11 @@ paths:
           schema:
             type: string
           description: Username uploading the csv
+        - in: query
+          name: status
+          schema:
+            type: boolean
+          description: Status for uploading.
```

Comment:
From this description I'm not sure how to use this parameter. Are you able to reword it to include expected effects? [Perhaps it doesn't perform the uploading, just checks that it can be done? I don't know]

Reply:
Will make this more clear in the description!